profile/src/test/resources/profileschema/0.2/mock-schema.json

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
i think our tests should run against the real schema, not this one, if they get out of sync that's going to be a bad thing